src/store/rocket.js

```diff
@@ -0,0 +1,74 @@
+import _ from 'lodash'
+import { WebClient } from '@slack/web-api'
+
+const state = () => {
+    return {employeeHeadshots: null, 
```

Comment:
There's a whole bunch of formatting issues with this file again too. Let's pair soon and figure out the root cause of this.